src/config/components/OverlayAdmin/OverlayAdmin.tsx

```diff
@@ -187,14 +210,68 @@ export default class OverlayAdmin extends React.Component<Props> {
 								</li>
 							) : null}
 							<li>
+								<SectionHeader>Bob's Buddy</SectionHeader>
+								<label>
+									<input
+										type="checkbox"
+										checked={bobsBuddyEnabled}
+										disabled={this.props.disabled}
+										onChange={this.setFeature(Feature.BOBSBUDDY)}
+									/>{" "}
+									Enable Bob's Buddy
+								</label>
+							</li>
+							{bobsBuddyEnabled ? (
+								<li>
+									<p style={{ fontStyle: "italic" }}>
+										Viewers can see your Bob's Buddy results. They can move and
+										hide the panel for themselves. Note: This feature requires
+										Bob's Buddy to be active on your Hearthstone Deck Tracker.
+										<br />
+									</p>
+
+									<Label as="span" style={{ marginBottom: "4px" }}>
+										When To Show Bob's Buddy Results:
+									</Label>
+									<Row width={"600px"}>
+										<label>
+											<input
+												type="radio"
+												checked={
+													whenToShowBobsBuddy === WhenToShowBobsBuddy.All
+												}
+												disabled={!bobsBuddyEnabled || this.props.disabled}
+												value={WhenToShowBobsBuddy.All}
+												onChange={this.changeWhenToShowBobsBuddy}
+											/>{" "}
```

Comment:
We should add a name property here and below just for good measure:
```suggestion
											<input
												type="radio"
												name="when_to_show_bobs_buddy"
												checked={
													whenToShowBobsBuddy === WhenToShowBobsBuddy.All
												}
												disabled={!bobsBuddyEnabled || this.props.disabled}
												value={WhenToShowBobsBuddy.All}
												onChange={this.changeWhenToShowBobsBuddy}
											/>{" "}
```